packages/image_picker/README.md

```diff
@@ -0,0 +1,36 @@
+# image_picker_tizen
+
+The Tizen implementation of [`image_picker`](https://github.com/flutter/plugins/tree/master/packages/image_picker).
+
+## Usage
+
+To use this plugin, add `image_picker` and `image_picker_tizen` as [dependencies in your pubspec.yaml file](https://flutter.io/platform-plugins/).
+
+## Example
+
+Import the library.
+
+``` dart
+import 'package:image_picker/image_picker.dart';
+```
+
+Then invoke the static `image_picker` method anywhere in your Dart code.
+
+``` dart
+final pickedFile = await picker.getImage(source: ImageSource.gallery);
+```
+
+## Limitations
+
+- This plugin is only supported on **Galaxy Watch** devices running Tizen 5.5 or later.
```

Comment:
How about adding this line:
```markdown
- The only API you can use is `getImage(source: ImageSource.gallery)`. You can't pick a video file (`getVideo()`) or pick a file from `ImageSource.camera`.
```

Reply:
done.

Reply:
Please append the line but do not remove the existing one. It's still necessary.

Reply:
done.

Reply:
Thank you!